tests/testthat/test-LoadReporterModule.R

```diff
@@ -106,3 +106,36 @@ testthat::test_that("report_load_srv - fail to load a reporter because of differ
 testthat::test_that("report_load_ui - returns a tagList", {
   checkmate::expect_multi_class(report_load_ui("sth"), c("shiny.tag.list", "shiny.tag"))
 })
+
+testthat::test_that("LoadReporterModule works", {
```

Comment:
This test looks like it was improvised. There is no such module. Reporter is not saved directly through saveRDS but there is a method in the Reporter object.
Please also be more specific in test name. Should rather be `report_load/ui/srv` restores reporter from saved bundle.